stable/spring-cloud-data-flow/README.md

```diff
@@ -146,10 +146,11 @@ The following tables list the configurable parameters and their default values.
 
 ### RabbitMQ Configuration
 
-| Parameter                  | Description                              | Default                   |
-| -------------------------- | ---------------------------------------- | ------------------------- |
-| rabbitmq.enabled           | Enable RabbitMQ as the middleware to use | true
-| rabbitmq.rabbitmqUsername  | RabbitMQ user name                       | user
+| Parameter                   | Description                                 | Default                   |
+| --------------------------- | ------------------------------------------- | ------------------------- |
+| rabbitmq.enabled            | Enable RabbitMQ as the middleware to use    | true
+| rabbitmq.rabbitmq.username  | RabbitMQ user name                          | user
```

Comment:
Seeing `rabbitmq` twice looks odd. Could we maybe change it to just `rabbitmq.username` and `rabbitmq.password` or is it a breaking change?

Reply:
unfortunately that wart came as part of upgrading... looks like some props were moved down a level (under `rabbitmq` block), so in the rabbit yaml itself its for example `rabbitmq.username`, we have a leading `rabbitmq` as thats how chart refs are done